src/clojure/clj_ldap/client.clj

```diff
@@ -355,6 +358,8 @@
    :trust-store     Only trust SSL certificates that are in this
                     JKS format file, optional, defaults to trusting all
                     certificates
+   :trust-managers  An optional TrustManager array to be used in place of
```

Comment:
These options are also described in the README.  Can you update that too?

Reply:
Updated